krun/results.py

```diff
@@ -183,17 +191,18 @@ def strip_results(self, key_spec):
     def append_exec_measurements(self, key, measurements):
         """Unpacks a measurements dict into the Results instance"""
 
+        # Has already been check earlier, but let's be defensive.
```

Comment:
Is it useful to do so? Too much checking can be confusing IME. But this is a judgement call, and I'm happy with whatever you decide here.